### DISCUSSION TITLE
Enhance the "clickability" of Expandable boxes

### DISCUSSION THREAD

**Discussion by SRyanPlanitar** [GitbookIO/community#220] [Feature Requests]
### What's your feature/idea?

It'd be nice to be able to click anywhere in the expandable box to expand the text, rather than having to click on the arrow.

### What's your desired outcome?

To be able to click anywhere in the Expand "block" feature to expand the text.

### What's the impact of this for you?

- [ ] Critical
- [ ] High
- [X] Medium
- [ ] Low

### Anything else?

Nothing else, this would just enhance the usability of the expandable boxes!

**Comment by taszap** [comment]
Related to #228

**Comment by SharronGitsIt** [comment]
I would add that it would also be nice to likewise be able to click anywhere to close. It's a tiny thing but I don't want my use of this feature to frustrate the users accessing the Guidebook we've implemented.  I'll post this elsewhere but I'd ask the same for inline images that can be clicked to view full size. In most apps you need only click anywhere in the image to reduce / close the image.

The easier we can make it for them, the better.
